ext/mysqli/tests/bug36802.phpt

```diff
@@ -24,11 +24,17 @@ Bug #36802 (crashes with with mysqli_set_charset())
 	/* following operations should work */
 	$x[2] = ($mysql->client_version > 0);
 	$x[3] = $mysql->errno;
-	$mysql->close();
+
+	try {
+	    $mysql->close();
```

Comment:
How did this go from no warning at all to an exception?

Reply:
It's a little bit delusive because I didn't really finish with the test fixes, and this test is still failing (https://travis-ci.org/php/php-src/jobs/633207641#L1847).

At the first sight, I thought that `$mysql->close();` is the source of the exception, but it's not the case so the try-catch block is not needed there. However, `$mysql->set_charset('utf8')` and `$mysql->query("SELECT 'foo' FROM DUAL")` both throw (but previously the warnings were suppressed by `@`).